<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-1FLO5NT" failures="2" tests="4" name="Test" time="23.288" errors="0" timestamp="2025-01-20T18:56:46 IST">
  <testcase classname="com.mk.tests.LoginTest" name="loginTest" time="1.51"/>
  <testcase classname="com.mk.tests.LoginTest" name="loginTest" time="1.536">
    <failure type="java.lang.AssertionError" message="
Expecting actual:
  &amp;quot;https://www.saucedemo.com/&amp;quot;
to contain:
  &amp;quot;inventory&amp;quot; ">
      <![CDATA[java.lang.AssertionError: 
Expecting actual:
  "https://www.saucedemo.com/"
to contain:
  "inventory" 
at com.mk.tests.LoginTest.loginTest(LoginTest.java:21)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1575)
... Removed 7 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
  <testcase classname="com.mk.tests.LoginTest" name="loginTest" time="1.219"/>
  <testcase classname="com.mk.tests.LoginTest" name="loginTest" time="0.602">
    <failure type="java.lang.AssertionError" message="
Expecting actual:
  &amp;quot;https://www.saucedemo.com/&amp;quot;
to contain:
  &amp;quot;inventory&amp;quot; ">
      <![CDATA[java.lang.AssertionError: 
Expecting actual:
  "https://www.saucedemo.com/"
to contain:
  "inventory" 
at com.mk.tests.LoginTest.loginTest(LoginTest.java:21)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1575)
... Removed 7 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
</testsuite> <!-- Test -->
